parquet/parquet.cpp

```diff
@@ -1497,6 +1497,10 @@ static void dissect_field(pq_writer *pw, lua_State *lua, pq_node *n, int16_t r,
       }
     }
     break;
+  case LUA_TLIGHTUSERDATA:
+    if (lua_touserdata(lua, -1) != NULL) {
+      break;
+    }
```

Comment:
comment the intended fall thru (GCC 7 is now making this a warning and it will fail the build)